ArduCopter/Attitude.cpp

```diff
@@ -57,7 +57,7 @@ void Copter::update_throttle_hover()
 
     // calc average throttle if we are in a level hover.  accounts for heli hover roll trim
     if (throttle > 0.0f && fabsf(vel_d_ms) < 0.6 &&
-        fabsf(ahrs.roll_sensor-attitude_control->get_roll_trim_cd()) < 500 && labs(ahrs.pitch_sensor) < 500) {
+        fabsf(ahrs.get_roll_rad() - attitude_control->get_roll_trim_rad()) < radians(5) && labs(ahrs.pitch_sensor) < 500) {
```

Comment:
I wonder if we should perhaps also change the pitch component to this?

fabss(ahrs.get_pitch_rad()) < radians(5)

optionally it might also be good to add some extra brackets before and after the "&&"